test/NuGet.Tests.Apex/NuGet.PackageManagement.UI.TestContract/ApexTestUIProject.cs

```diff
@@ -57,6 +56,19 @@ public void Search(string searchText)
             UIInvoke(() => _packageManagerControl.Search(searchText));
         }
 
+        public bool SearchPackageVerification(string tabName, string packageId, string packageVersion = null)
```

Comment:
This could use a better name. The name should talk about what it does, not why it does. 
`Search` is a why, as it's what you're testing, but realistically, you're asserting the top package in the list.

`VerifyFirstPackageOnTab` or something like that.

Reply:
Thanks for the suggestion, I've updated the name. Please help to review it again. Thanks.